### ISSUE TITLE
请问怎么从硬盘中加载图片集进行训练呢？

### ISSUE THREAD

**Issue by JimXu1989** [PaddlePaddle/Paddle#30565]
我使用的是paddle2.0rc，示例中给的都是加载自带的数据集，我想从硬盘中读取数据集，请问要怎么操作呢？谢谢

**Comment by lfchener** [comment]
https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/io/DataLoader_cn.html#dataloader
可以用 DataLoader 加载自定义的数据集

**Comment by JimXu1989** [comment]
> https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/io/DataLoader_cn.html#dataloader
> 可以用 DataLoader 加载自定义的数据集

请问有相关的例程吗？我加载图片训练总是失败，而且
train_loader = paddle.batch(paddle.dataset.mnist.train(), batch_size=10)
和
train_loader = DataLoader(dataset,batch_size=BATCH_SIZE, shuffle=True, drop_last=True, num_workers=2)
这两种方法产生的loader并不是一种类型

还有，有没有能自动把数据集拆分成训练集和测试集的函数？

感谢
